subsys/shell/Kconfig

```diff
@@ -251,6 +251,14 @@ config SHELL_AUTOSTART
 	help
 	  If enabled, shell will be automatically started.
 
+config SHELL_CMDS_RETURN_VALUE
```

Comment:
This option is not used anywhere, see comment below.

Reply:
Edit made and retested.